### DISCUSSION TITLE
Assetto Corsa Competizione

### DISCUSSION THREAD

**Discussion by steevivo** [Sergih28/setup-comparator#37] [General]
Hello,
Where is the link for Assetto Corsa Competizione please ? your job is awesome

**Comment by Sergih28** [comment]
> Hello, Where is the link for Assetto Corsa Competizione please ? your job is awesome

Hi @steevivo!
It just works with rFactor 2 setups. I haven't been working on this for a while and I don't think I will continue with this project.

Feel free to make the ACC functionality yourself if you want to and it can be merged afterwards! 👍
